### PR TITLE
Feature/routing template

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Add routing template
- https://pxblue-temp.web.app

### CODE REVIEW

**Comment by emclaug2** [comment]
I'm not entirely sold on the folder structure; I just wanted to move all logic out of the app component.

File: src/app/routing/template/navigation/navigation.component.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                    }
+                    default: {
+                        this.toolbarTitle = '';
+                    }
+                }
+            }
+        });
+    }
+
+    getVariant(): DrawerLayoutVariantType {
```
Comment (emclaug2) [inline src/app/routing/template/navigation/navigation.component.ts:87]: What is this doing?

**Comment by joebochill** [comment]
I think this looks reasonable to me. Is there an alpha version with the latest updates on NPM to test it against the CLI?

**Comment by emclaug2** [comment]
> I think this looks reasonable to me. Is there an alpha version with the latest updates on NPM to test it against the CLI?

I just published a new non-alpha alpha version so the pxb cli will detect it.   Should work now.

**Comment by emclaug2** [comment]
I'm not a fan of the drawer slide-in animation on app startup, but that's a bug in our drawer component.

**Comment by joebochill** [comment]
Looks good, runs as expected. I think we may want to update the CLI to run the `yarn lint --fix` and `yarn prettier` after setting things up (if they selected to use our profile) to give the user a clean starting point, but we can make a new issue for that.

**Review by joebochill — APPROVED** [review]

**Pull request merged**